content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -407,6 +407,11 @@ sudo apt-get update && sudo apt-get install -y \
   docker-ce-cli=5:19.03.11~3-0~ubuntu-$(lsb_release -cs)
 ```
 
+```shell
+## Create /etc/docker
+sudo mkdir /etc/docker
```

Comment:
```suggestion
# Ensure that /etc/docker exists
sudo mkdir -p /etc/docker
```
will work even if /etc/docker already exists. How about this?

Reply:
Well, you guys are really nitpicky.... If you follow the instructions, the folder isn't there yet. That's what it is all about, right? If the command fails, it has no purpose anyway. Also, the suggested commit is an exact copy of the "CentOS/RHEL 7.4+" tab. So doing it this way makes it more consistent. 

p.s. Docker runtimes are deprecates anyway by now, right?

Reply:
I know in a script / control loop I'd handle both cases, and that's the way I usually write code.

We have _not_ deprecated use of Docker with Kubernetes. We did deprecate the built-in `dockershim` though.

Anyway, it seems like the existing page has a mix of `mkdir -p` and plain `mkdir`.